src/components/HeroPractise/HeroPractise.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  HeroButton,
+  HeroButtonContainer,
+  HeroButtonText,
+  HeroContainer,
+  HeroImage,
+  HeroLink,
+  HeroText,
+  HeroTitle,
+  TextContainer,
+} from './styled';
+
+type HeroPractiseProps = {
+  link: string;
```

Comment:
Maybe rename this to linkText (?) to show more clearly that this props sets the HeroPractice link text/label?